src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.html

```diff
@@ -0,0 +1,158 @@
+<template>
+  <pcard if.bind="viewMode === 'edit'">
+    <pform-input
+      helper-message.bind="tokenInfoLoading ? 'Searching token address details...': ''"
+      input-reference.bind="addressInput"
```

Comment:
Nit: 
(Best in another PR) (Fine with ignoring this)
The word "reference" is a bit ambiguous in the context of an Aurelia View.
Because one uses 
`<my-ele ref="xyz">`
more than
`<my-ele view-model.ref="abc">`

--> I'd suggest to use "reference" when we refer to an HTMLElement.
In other cases, we make it more explicit like:
```suggestion
      input-view-model.bind="addressInputViewModel"
```

Reply:
This is a bigger change since there are a few more pages that uses this. So yes, we need a different PR for it. (feel free to create an issue for it if you want )